modules/ref/pages/client-settings.adoc

```diff
@@ -47,6 +47,17 @@ a custom strategy is fairly advanced, the SDK ships with a best effort retry str
 out of the box (BestEffortRetryStrategy). This default will retry the operation until it
 either succeeds or the maximum request lifetime is reached.
 
+Name: *OutOfOrderEnabled*::
```

Comment:
This is very much not a thing. Out of order can only be controlled via the query string and it's `unordered_execution_enabled`

Reply:
Thanks